packages/cashier/src/modules/cashier-onboarding/components/cashier-onboarding-fiat-card/cashier-onboarding-fiat-card.tsx

```diff
@@ -42,21 +42,25 @@ const CashierOnboardingFiatCard: React.FC = observer(() => {
     };
 
     return (
-        <CashierOnboardingCard
-            title={localize('Deposit via bank wire, credit card, and e-wallet')}
-            description={localize('Deposit via the following payment methods:')}
-            depositCategory='fiat'
-            onClick={is_dialog_visible ? undefined : onClick}
-        >
-            <CashierOnboardingIconMarquee icons={icons.map(icon => `${icon}${is_dark_mode_on ? 'Dark' : 'Light'}`)} />
+        <React.Fragment>
+            <CashierOnboardingCard
+                title={localize('Deposit via bank wire, credit card, and e-wallet')}
+                description={localize('Deposit via the following payment methods:')}
+                depositCategory='fiat'
+                onClick={is_dialog_visible ? undefined : onClick}
+            >
+                <CashierOnboardingIconMarquee
+                    icons={icons.map(icon => `${icon}${is_dark_mode_on ? 'Dark' : 'Light'}`)}
+                />
+            </CashierOnboardingCard>
             {can_switch_to_fiat_account && (
```

Comment:
Here we are extracting `SwitchToFiatAccountDialog` from  `CashierOnboardingCard` because button onClick events of AccountDialogSwitcher propagating to `CashierOnboardingCard` because previously `SwitchToFiatAccountDialog` were children inside `CashierOnboardingCard`.
IMHO this is the simplest way to fix it because in other case we need to modify Dialog component and stop propagation there. Since Dialog component is using across the whole app, i decided to extract `SwitchToFiatAccountDialog` instead.